lib/include/openamp/virtio_ring.h

```diff
@@ -43,13 +46,14 @@ struct vring_desc {
 	uint16_t flags;
 	/* We chain unused descriptors via this, too. */
 	uint16_t next;
-};
+}METAL_PACKED_END;
 
+METAL_PACKED_BEGIN
 struct vring_avail {
 	uint16_t flags;
 	uint16_t idx;
 	uint16_t ring[0];
-};
+}METAL_PACKED_END;
```

Comment:
Today no warning on other structures but could you to extend patch for vring_used_elem and vring_used and vring  structure.
thanks

Reply:
Done